### PR TITLE
fixed some extra stuff that was missed

### PR DESCRIPTION
**Pull request by developer9998**
so, what did i do?

fixed the name tab sometimes displaying a wrong name
before
image
after
image

fixed your colour not changing if you weren't in a room
before
https://user-images.githubusercontent.com/81720436/183551593-2ec95f0f-3821-4aef-b555-ad3c3cd06c27.mp4
after
https://user-images.githubusercontent.com/81720436/183551605-efe47455-0434-40f2-9f0b-a3fda5bab8fc.mp4

updated the credits tab                                  
couldn't get any screenshots/videos, sorry!

### CODE REVIEW

File: ComputerInterface/Views/GameSettings/JoinRoomView.cs [hunk 1]
```diff
@@ -60,6 +60,166 @@ public void Redraw()
...
+        }
+
+        // Gets connection state if that wasn't obvious
+        private PhotonNetworkController.ConnectionState GetConnectionState()
+        {
+            return (PhotonNetworkController.ConnectionState)Traverse.Create(PhotonNetworkController.Instance).Field("currentState").GetValue();
+        }
+    }
+}
+=======
```
Comment (Graicc) [inline ComputerInterface/Views/GameSettings/JoinRoomView.cs:160]: You didn't finish the merge

File: ComputerInterface/Views/GameSettings/JoinRoomViewCallbacks.cs [hunk 2]
```diff
@@ -19,3 +19,25 @@ class JoinRoomViewCallbacks : MonoBehaviourPunCallbacks
         public override void OnConnectedToMaster() => view.Redraw();
     }
 }
+=======
```
Comment (Graicc) [inline ComputerInterface/Views/GameSettings/JoinRoomViewCallbacks.cs:22]: Same here

File: ComputerInterface/Views/GameSettings/JoinRoomViewCallbacks.cs [hunk 3]
```diff
@@ -19,3 +19,25 @@ class JoinRoomViewCallbacks : MonoBehaviourPunCallbacks
...
+        public JoinRoomView view;
+
+        public override void OnJoinedRoom() => view.Redraw();
+        public override void OnCreatedRoom() => view.Redraw();
+        public override void OnLeftRoom() => view.Redraw();
+        public override void OnDisconnected(DisconnectCause cause) => view.Redraw();
+        public override void OnJoinRoomFailed(short returnCode, string message) => view.Redraw();
+        public override void OnJoinRandomFailed(short returnCode, string message) => view.Redraw();
+        public override void OnCreateRoomFailed(short returnCode, string message) => view.Redraw();
+        // public override void OnSchoolFailed(short returnCode, string message) => view.Redraw();
```
Comment (Graicc) [inline ComputerInterface/Views/GameSettings/JoinRoomViewCallbacks.cs:39]: OnSchoolFailed?

File: ComputerInterface/Views/GameSettings/CreditsView.cs [hunk 4]
```diff
@@ -48,6 +48,18 @@ private void Redraw()
...
+            str.AppendLine("Credits").AppendLine()
+                .AppendLine("Game by Kerestell").AppendLine();
+
+            str.AppendLine("\"Monke Need To Swing\"")
+                .AppendLine("Composed by Stunshine")
+                .AppendLine("Produced by Audiopfeil & Owlobe").AppendLine();
+
+            str.AppendLine("\"Cave Wave\" & \"Campfire\"")
+                .AppendLine("Composed by Stunshine")
+				.AppendLine("Sound design by David Anderson Kirk");
```
Comment (Graicc) [inline ComputerInterface/Views/GameSettings/CreditsView.cs:62]: Is this not already covered in the pages above?

**Review by Graicc — CHANGES_REQUESTED** [review]

File: ComputerInterface/Views/GameSettings/JoinRoomViewCallbacks.cs (hunk 3 above)
Comment (developer9998) [reply ComputerInterface/Views/GameSettings/JoinRoomViewCallbacks.cs:39]: i didnt mess with that lol

File: ComputerInterface/Views/GameSettings/JoinRoomView.cs (hunk 1 above)
Comment (developer9998) [reply ComputerInterface/Views/GameSettings/JoinRoomView.cs:160]: what now i hardly know anything about github

File: ComputerInterface/Views/GameSettings/CreditsView.cs (hunk 4 above)
Comment (developer9998) [reply ComputerInterface/Views/GameSettings/CreditsView.cs:62]: github being stupid
i think

**Pull request closed without merging**